### PR TITLE
notty_async.0.1

### PR DESCRIPTION
**Pull request by yminsky**
Trying again, this time against opam 1.2, and not using the opam-publish script!

### CODE REVIEW

File: packages/notty_async/notty_async.0.1/opam [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  ["jbuilder" "subst"] {pinned}
+  ["jbuilder" "build" "-p" name "-j" jobs]
+]
+
+depends: [
+  "jbuilder" {build & >="1.0+beta20"}
+  "notty"    {>= "0.2.1"}
+  "async"    {>= "v0.11.0"}
+  "base"     {>= "v0.11.0"}
+  "ocaml"    {>= "4.06.0"}
```
Comment (dra27) [inline packages/notty_async/notty_async.0.1/opam]: `ocaml` is a 2.0.0-only package, so this wants to be deleted...

File: packages/notty_async/notty_async.0.1/opam [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+]
```
Comment (dra27) [inline packages/notty_async/notty_async.0.1/opam]: ... and below here add:
```
available: ocaml-version >= "4.06.0"
```

**Comment by yminsky** [comment]
Better, now?

**Comment by kit-ty-kate** [comment]
The Alpine failure is caused by https://github.com/janestreet/core/issues/109. Otherwise all seem good. Thanks!

**Pull request merged**